### PR TITLE
アドオンマネージャーの国際化 (Support i18n)

### PR DESCRIPTION
**Pull request by kajikum**
[angular-translate](https://angular-translate.github.io/)を用いた国際化

選択している言語は、localStorage or Cookieに保存。
各言語の翻訳情報は、`{ISO639の言語コード（2文字）}.json`という名前で管理

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,12 +2,12 @@
 <html lang="ja" data-ng-app="app">
 	<head>
 		<meta charset="UTF-8">
-		<title>Tree of Savior Addon Manager</title>
+		<title>{{ 'TITLE' | translate }}</title>
```
Comment (kajikum) [inline index.html]: とりあえずタイトルだけ実装。  
ここは変えなくてもいいんだろうけど、目につきやすいところだけを変えてみた

**Comment by shimonami** [comment]
Hello guys!
Whats the progress on this PR?
I was thinking of working on a multi-language support, and I've seen this open PR.

Will this be merged?

**Comment by writ312** [comment]
@shimonami 
I do not edit this repository
Because I got tired.
If you want to PR in this repository, I invite you to that organization.

**Comment by shimonami** [comment]
@writ312 thanks for your invite!
actually this code was already merged, is it working as expected or I need to test and translate?

**Comment by kajikum** [comment]
Hi, @shimonami.

That's very kind of you.
But it seems that the latest code hasn't been pushed yet, so I don't think we need to carefully test that.

**Pull request merged**